news/8857.bugfix.rst

```diff
@@ -0,0 +1,7 @@
+(Windows) Fix Qt run-time hooks failing to add the top-level application
```

Comment:
Not familiar with the project’s terminology here, so this comment might be wrong but the issue I encountered was with the `_internal` folder, not the top-level directory that contained the executable.

Reply:
"Top-level application directory" is what `sys._MEIPASS` points to; in onedir builds, this is the `_internal` directory (unless you configure it otherwise) and in onefile builds, it is the temporary directory where application unpacked itself.